examples/plot_derivative.py

```diff
@@ -0,0 +1,79 @@
+"""
+Derivatives
+===========
+This example demonstrates how to use a set of derivative operators, namely
+:py:class:`pylops_mpi.basicoperators.MPIFirstDerivative` and
+:py:class:`pylops_mpi.basicoperators.MPISecondDerivative`.
```

Comment:
I would add this sentence, as this is the point we need these operators.

``We will be focusing here on the case where the input array `x` is assumed to be an n-dimensional distributed array and the derivative is applied over the first axis (`axis==0`). Since the array is distributed over multiple processes, the derivative operators must take care of applying the derivatives across the edges using information from the previous/next process, using so-called ghost cells.`
``

Reply:
Added this sentence.